vowpalwabbit/offset_tree_internal.cc

```diff
@@ -0,0 +1,138 @@
+#include "offset_tree_internal.h"
+#include "vw_exception.h"
```

Comment:
Why is it worthwhile to have an internal/not distinction?  Spreading a single reduction across multiple files seems undesirable by default?

Reply:
Will introduce a header for min-depth tree and merge these into offset_tree.cc

Reply:
Done.